README.md

```diff
@@ -7,61 +7,168 @@
 
 * Explain that by making parts of their code *conditional*, they can execute different pieces of code based on certain conditions.
 * Explain that an *if* statement will execute a set of statements **only** if that condition is *true*. 
+* Explain that an *if* statement can provide an alternative set of statements, known as an *else clause*, for situations when the *if* condition is *false*.
+* Create/chain multiple *if-else* statements together to consider additional clauses:
+
+
+
+## Outline / Notes
+
+In the previous lesson we covered Boolean values (namely ````true```` and ````false````) along with the various operations we can perform that result in a Boolean value (namely ````<, ==, <=, >, >=, !=````).
+For example:
+
+````Swift
+3 < 5    // false
+12 >= 12  // true
+let greedoFiredFirst  // true
+! greedoFiredFirst  // false
+````
+
+At this point you may find Booleans interesting but are wondering what they're actually good for.  If that's the case then you're in luck because that's *exactly* what this lesson is about.
 
-```swift
-var age = 19
+So far you've covered computer code that's of the form of directives and declarations; that is, "do this".  For example:
 
-if age < 21 {
-    print("You're not allowed to drink!")
+````Swift
+print("hello")   // hello
+// directive to print "hello"
+
+var x = 5 * 8
+// declaration of variable x
+
+let i = 12 >= 9
+// declaration of constant i
+````
+
+Most programs, however, need an additional component, and that component is Conditionals.  Conditionals are how you tell the program to take different behavior based upon conditions or results.  The simplest form of Conditional is just to do something if an expression is true.  We can express "if class is hard then study more" like this:
+
+````Swift
+if classIsHard {
```

Comment:
I like this example a LOT. I think we can be slightly more explicit here in doing something like this.

---

```swift
func studyMore() {
    print("Study 📖")
}

var classIsHard = true

if classIsHard {
    studyMore()
}

// prints "Study 📖"
```

Here we created a function named `studyMore` which takes no arguments and doesn't return anything. When this function is called, it will print 'Study 📖' to console. Following that, we're creating a variable named `classIsHard` of type `Bool` with a value of `true`. The next part should be self-evident in that it reads like a sentence. We're using an `if` statement. If `classIsHard` is `true` (which it is) then we should execute the code in the following `{ }` curly braces. The code in the `{ }` curly braces following that `if` statement is a call to the function `studyMore()`. As it stands, 'Study 📖' will print to console because `classIsHard` has a value of `true`. But what if we change `classIsHard` to `false`? If we did, then the `studyMore()` function will **not** be called and nothing will print to console. The compiler will continue on its merry way because the `if` statement would not have evaluated to `true`, thus it would not have entered those `{ }` curly braces.